src/Semantic/Util.hs

```diff
@@ -76,54 +64,29 @@ justEvaluating
   . runAddressError
   . runValueError
 
-type FileEvaluator err syntax =
-  [FilePath]
-  -> IO
-       ( Heap Precise Precise (Value (Quieterm (Sum syntax) Loc) Precise),
-       ( ScopeGraph Precise
-       , Either (SomeError (Sum err))
-                (ModuleTable (Module (ModuleResult Precise (Value (Quieterm (Sum syntax) Loc) Precise))))))
-
-evalGoProject :: FileEvaluator _ Language.Go.Assignment.Syntax
-evalGoProject = justEvaluating <=< evaluateProject (Proxy :: Proxy 'Language.Go) goParser
-
-evalRubyProject :: FileEvaluator _ Language.Ruby.Assignment.Syntax
-evalRubyProject = justEvaluating <=< evaluateProject (Proxy @'Language.Ruby)               rubyParser
-
-evalPHPProject :: FileEvaluator _ Language.PHP.Assignment.Syntax
-evalPHPProject  = justEvaluating <=< evaluateProject (Proxy :: Proxy 'Language.PHP)        phpParser
-
-evalPythonProject :: FileEvaluator _ Language.Python.Assignment.Syntax
-evalPythonProject = justEvaluating <=< evaluateProject (Proxy :: Proxy 'Language.Python)     pythonParser
-
-evalTypeScriptProject :: FileEvaluator _ Language.TypeScript.Assignment.Syntax
-evalTypeScriptProject = justEvaluating <=< evaluateProject (Proxy :: Proxy 'Language.TypeScript) typescriptParser
-
-evaluateProject proxy parser paths = withOptions debugOptions $ \ config logger statter ->
-  evaluateProject' (TaskSession config "-" False logger statter) proxy parser paths
```

Comment:
The net result of deleting all of this is compilation has gone from ~9s to ~0.3s (IIRC).